src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -184,9 +184,15 @@ public void saveCountyCourtJudgment(String authorisation, Claim claim, CountyCou
         caseRepository.saveCountyCourtJudgment(authorisation, claim, countyCourtJudgment);
     }
 
-    public void saveDefendantResponse(long claimId, String defendantId, String defendantEmail, Response response) {
+    public void saveDefendantResponse(
+        Claim claim,
+        String defendantId,
+        String defendantEmail,
+        Response response,
+        String authorization
+    ) {
         // When this is saved in CCD ensure a Forbidden response is returned to the client if they
```

Comment:
can you test this and remove the comment if true? it should work after I mapped the feign exception